shell/platform/darwin/ios/framework/Source/FlutterViewControllerTest.mm

```diff
@@ -124,6 +124,12 @@ - (void)handlePressEvent:(FlutterUIPressProxy*)press
 - (void)scrollEvent:(UIPanGestureRecognizer*)recognizer;
 - (void)updateViewportMetrics;
 - (void)onUserSettingsChanged:(NSNotification*)notification;
+- (void)applicationWillTerminate:(NSNotification*)notification;
+- (void)goToApplicationLifecycle:(nonnull NSString*)state;
+- (void)keyboardWillChangeFrame:(NSNotification*)notification;
+- (void)startKeyBoardAnimation:(NSTimeInterval)duration;
+- (void)ensureViewportMetricsIsCorrect;
+- (void)invalidateDisplayLink;
```

Comment:
This was a merge conflict with #29281 which we don't need here.  We do need `applicationWillTerminate` and `goToApplicationLifecycle`.
```suggestion
```